src/main/java/net/geforcemods/securitycraft/datagen/BlockTagGenerator.java

```diff
@@ -278,7 +324,11 @@ protected void registerTags()
 				SCContent.REINFORCED_RED_NETHER_BRICK_WALL.get(),
 				SCContent.REINFORCED_SANDSTONE_WALL.get(),
 				SCContent.REINFORCED_END_STONE_BRICK_WALL.get(),
-				SCContent.REINFORCED_DIORITE_WALL.get());
+				SCContent.REINFORCED_DIORITE_WALL.get(),
+				SCContent.REINFORCED_BLACKSTONE_WALL.get(),
+				SCContent.REINFORCED_POLISHED_BLACKSTONE_WALL.get(),
+				SCContent.REINFORCED_POLISHED_BLACKSTONE_BRICK_WALL.get());
+		getOrCreateBuilder(BlockTags.WITHER_SUMMON_BASE_BLOCKS).add(SCContent.REINFORCED_SOUL_SOIL.get());

```

Comment:
Reinforced Soul Sand should also be added to this.

Reply:
See review comment above please